app/Http/Controllers/VacationLimitController.php

```diff
@@ -25,6 +27,7 @@ public function edit(YearPeriodRetriever $yearPeriodRetriever, UserVacationStats
 
         $limits = $yearPeriod
             ->vacationLimits()
+            ->whereRelation("user", fn(Builder $query): Builder => $query->withTrashed($request->user()->hasPermissionTo("showInactiveUsers")))
```

Comment:
As this is really common thing in codebase, maybe `$user->canSeeInactiveUsers` would be nice helper?

Reply:
changed